src/style.css

```diff
@@ -0,0 +1,132 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  background-color: #e5e5e5;
+}
+
+body,
+p {
+  margin: 0;
+
+  --white-for-card: #fff;
+  --blue-for-card: #00acdc;
+  --black--for-card: #060b35;
+  --grey--for-card: #616070;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card {
```

Comment:
1.create the separate files for each block
2. BEM-block shouldn't affect external geometry (in the form of padding, borders, dimensions) and positioning